packages/twenty-front/src/modules/settings/security/types/AuthProviders.type.ts

```diff
@@ -0,0 +1,4 @@
+export type AuthProvidersKeys =
```

Comment:
GOOGLE
MICROSOFT

isGoogleAuthEnabled
isMicrosoftAuthEnabled

map(authEnablekyey: AuthProvidersKeys): GOOGLE | MICROSOFT (

)

Reply:
not mandatory but I would have "avoided the complexity like this"